tests/phpunit/integration/Core/Conversion_Tracking/Conversion_Event_Providers/Easy_Digital_DownloadsTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * @package   Google\Site_Kit
+ * @copyright 2024 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Tests\Core\Conversion_Tracking\Conversion_Event_Providers;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Assets\Script;
+use Google\Site_Kit\Core\Conversion_Tracking\Conversion_Event_Providers\Easy_Digital_Downloads;
+use Google\Site_Kit\Tests\TestCase;
+
+class Easy_Digital_DownloadsTest extends TestCase {
+
+	/**
+	 * Easy_Digital_Downloads instance.
+	 *
+	 * @var Easy_Digital_Downloads
+	 */
+	private $contactform;
+
+	public function set_up() {
+		parent::set_up();
+		$this->contactform = new Easy_Digital_Downloads( new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE ) );
+	}
+
+	public static function tear_down_after_class() {
+		parent::tear_down_after_class();
+
+		if ( function_exists( 'runkit7_constant_remove' ) ) {
+			runkit7_constant_remove( 'EDD_VERSION' );
+		} elseif ( function_exists( 'runkit_constant_remove' ) ) {
+			runkit_constant_remove( 'EDD_VERSION' );
+		}
+	}
```

Comment:
Raising because other tests started failing for me locally as I don't have this extension installed. I'm not a big fan of it TBH, but there are a few things to note here.

- We only have 1 test defining the constant, so rather than using this we can use the `@runInSeparateProcess` annotation so that the constant can be defined for real without affecting other tests (e.g. `\Google\Tests\Core\Conversion_Tracking\Conversion_Event_Providers\WooCommerceTest::test_is_active`)
- Usually tear down like this we want to do after every test but this runs once after all tests in the class